src/components/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import icon from "../assets/images/night.png";
+import style from "./Header.module.scss";
+import iconMoon from "../assets/images/moon.png";
```

Comment:
Is it possible to have the same images as SVG?

And I'm not sure about the naming. At least the first one shouldn't be just `icon`.

Reply:
The problem is in template we see an empty moon and a full moon. I'm not sure I knew how to change them.
If you can give me a little advance  I will try
name changed :)

Reply:
I'm sure there are a lot of svg icons in the internet. There are libraries like react-icons, or websites like flaticon.com where you can find something similar and just grab svg code from there.

Reply:
done